testsuite/MDAnalysisTests/coordinates/test_mol2.py

```diff
@@ -214,4 +214,7 @@ def test_mol2_universe_write(tmpdir):
         u2 = mda.Universe(outfile)
 
         assert_almost_equal(u.atoms.positions, u2.atoms.positions)
-        assert_almost_equal(u.dimensions, u2.dimensions)
+        if u.dimensions is None:
+            assert u2.dimensions is None
+        else:
+            assert_almost_equal(u.dimensions, u2.dimensions)
```

Comment:
Is this ever triggered? From what I understand `dimensions` is always `None` right? If we're not sure, can we raise an issue for this please?